<launch>

	  <group ns="camera1">

   		<node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
	      <param name="camera_name" value="overhead_camera"/>
	      <param name="camera_info_url" value="package://darknet_ros/config/overhead_camera.yaml" />      
	      <param name="video_device" value="/dev/video0" />
	      <param name="image_width" value="640" />
	      <param name="image_height" value="480" />
	      <param name="framerate" value="10" />
	      <param name="pixel_format" value="yuyv" />
	      <param name="camera_frame_id" value="usb_cam" />
	      <param name="io_method" value="mmap"/>
	    </node>



	  </group>

	  


</launch>
